x-pack/plugins/search_playground/public/hooks/use_load_connectors.ts

```diff
@@ -52,7 +52,8 @@ const connectorTypeToLLM: Array<{
     actionId: OPENAI_CONNECTOR_ID,
     match: (connector) =>
       connector.actionTypeId === OPENAI_CONNECTOR_ID &&
-      (connector as OpenAIConnector).config.apiProvider === OpenAiProviderType.OpenAi,
+      ((connector as OpenAIConnector)?.config?.apiProvider === OpenAiProviderType.OpenAi ||
+        !!connector.isPreconfigured),
```

Comment:
I guess `connector` can never be `null` right?

Reply:
should never be null.

Reply:
so looks like if it is preconfigured, you will guess that it is OpenAI, not azure? Have you tested with Azure, are there any side effects to treating it as OpenAI?

Reply:
yep you're correct and might need a bigger refactor. For now il spin up a ticket for this and merge this one to prevent errors stopping devs using it.